### DISCUSSION TITLE
Aria-label on components?!

### DISCUSSION THREAD

**Discussion by vitalijalbu** [rsuite/rsuite#2790] [Ideas]
Hi guys, it's possible to add aria-labels on all components? It makes components more user-click-friendly and more trustable when clicking.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

**Comment by SevenOutman** [comment]
@vitalijalbu Which components you're using failed to apply `aria-label` attribute?

**Reply by vitalijalbu** [reply]
none, i mean could you add them by default?

**Reply by SevenOutman** [reply]
The label on an element should describe what a that element stands for, which only the user of our components (author of applications) would know.

Talking about "default" labels, I don't think most components would have one that really makes sense. Do you have some suggestions of what default labels should be added on what components?
